pkg/asset/cluster/azure/azure.go

```diff
@@ -27,6 +27,8 @@ func Metadata(config *types.InstallConfig) *azure.Metadata {
 		Region:                      config.Platform.Azure.Region,
 		ResourceGroupName:           config.Azure.ResourceGroupName,
 		BaseDomainResourceGroupName: config.Azure.BaseDomainResourceGroupName,
+		ZoneName:                    config.BaseDomain,
+		APIURL:                      fmt.Sprintf("api.%s", config.ObjectMeta.Name),
```

Comment:
I don't think we should save the APIURL to the metadata. We already have the cluster name in the metadata, so whenever you need the api url in the destory code just do the Sprintf there.

Reply:
This did not get fixed in your more recent pushes. WDYT?

Reply:
Ah forgot about that. Fixed the code and added a *.apps delete too.